manifests/config.pp

```diff
@@ -70,6 +70,71 @@
     mode   => '0700',
     before => File['/etc/ipsec.conf']
   }
+
```

Comment:
This looks great, but I think we need to probably pull this out into a separate subclass if possible. I haven't dug all the way into the code but making this part of the default configuration seems a bit heavy handed if users don't necessarily want it.